src/panels/lovelace/components/hui-warning.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  property,
+  CSSResult,
+  css,
+} from "lit-element";
+
+export class HuiWarning extends LitElement {
+  @property() public warning?: string;
+  @property() public entity?: string;
+
+  public getCardSize(): number {
+    return 1;
+  }
+
+  protected render(): TemplateResult | void {
+    return html`
+      ${this.warning || "Entity not available"}: ${this.entity || ""}
+    `;
+  }
+
+  static get styles(): CSSResult {
+    return css`
+      :host {
+        display: block;
+        color: black;
+        background-color: yellow;
```

Comment:
This yellow is ugly. Let's use #FCE588

![image](https://user-images.githubusercontent.com/1444314/52909918-9a55f400-3244-11e9-9877-d9f9d3bbfc32.png)